xgboost_ray/main.py

```diff
@@ -282,6 +284,20 @@ def _cleanup(checkpoint_prefix: str, checkpoint_path: str, num_actors: int):
             os.remove(checkpoint_file)
 
 
+def _shutdown(remote_workers: List[ActorHandle], force: bool = False):
+    if force:
+        for worker in remote_workers:
+            logger.debug(f"Killing worker {worker}.")
```

Comment:
I think one log is enough (otherwise this might flood the debug logs). Something like `logger.debug(f"Killing {len(remote_workers)} workers")`

Reply:
Done!